models/endoscopic_tool_segmentation/docs/README.md

```diff
@@ -1,10 +1,17 @@
 # Model Overview
-A pre-trained model for the endoscopic tool segmentation task and is trained using a flexible unet structure with an efficient-b2 [1] as the backbone and a UNet architecture [2] as the decoder. Datasets use private samples from [Activ Surgical](https://www.activsurgical.com/).
+A pre-trained model for the endoscopic tool segmentation task, trained using a flexible unet structure with an efficientnet-b2 [1] as the backbone and a UNet architecture [2] as the decoder. Datasets use private samples from [Activ Surgical](https://www.activsurgical.com/).
 
-The [PyTorch model](https://drive.google.com/file/d/19yS3t2oLBiB7wT-qeQ82da95VJs_vzRK/view?usp=share_link) and [torchscript model](https://drive.google.com/file/d/1cDZ3Jr7mhpzdzaFyz8yHNowH8k0T1VZz/view?usp=share_link) are shared in google drive. Details can be found in large_files.yml file. Modify the "bundle_root" parameter specified in configs/train.json and configs/inference.json to reflect where models are downloaded. Expected directory path to place downloaded models is "models/" under "bundle_root".
+The [PyTorch model](https://drive.google.com/file/d/1I7UtWDKDEcezMqYiA-i_hsRTCrvWwJ61/view?usp=sharing) and [torchscript model](https://drive.google.com/file/d/1e_wYd1HjJQ0dz_HKdbthRcMOyUL02aLG/view?usp=sharing) are shared in google drive. Details can be found in large_files.yml file. Modify the "bundle_root" parameter specified in configs/train.json and configs/inference.json to reflect where models are downloaded. Expected directory path to place downloaded models is "models/" under "bundle_root".
 
 ![image](https://developer.download.nvidia.com/assets/Clara/Images/monai_endoscopic_tool_segmentation_workflow.png)
 
+## Pre-trained weights
+A pre-trained encoder weights would benefit the training accuracy. Either the [ImageNet](https://ieeexplore.ieee.org/document/5206848) pre-trained or a local pre-trained encoder weights could be loaded to the model for the segmentation training. Whether to use the ImageNet pre-trained weights is controlled by the `use_imagenet_pretrain` parameter in the `configs/train.json` file. If using the local pre-trained weights, please update the `load_path`  parameter of the `CheckpointLoader` in the `configs/train.json` file with the local weights path.
+
+- We trained a pre-trained encoder weights using some internal data.
+
+- The [EfficientNet-PyTorch repo](https://github.com/lukemelas/EfficientNet-PyTorch) provides the ImageNet pre-trained encoder weights but for non-commercial use. The ImageNet pre-trained encoder weights would be loaded for training, if setting the `use_imagenet_pretrain` parameter to `True`. Each user is responsible for checking the content of the models/datasets and the applicable licenses and determining if suitable for the intended use.
```

Comment:
duplicate information between this line and https://github.com/Project-MONAI/model-zoo/pull/364/files#diff-1d6658e5d0f1616da1655e5be3b8a383c0c6a7e1e4782fd6210850c48444f00aR9
I will update the readme to address it